example-new-site/design4-home.html

```
@@ -0,0 +1,90 @@
+<!-- ABOUTME: Standalone home page extracted from Design 4 of the Harper Blog Redesign. -->
+<!-- ABOUTME: Preserves the original visual markup, theme toggle, and image-slot dependencies. -->
+<!DOCTYPE html>
+<html>
```

HTMLHint (1.9.2)

[warning] An lang attribute must be present on <html> elements.

(html-lang-require)